app/uk/gov/hmrc/gform/sharedmodel/formtemplate/SummarySection.scala

```diff
@@ -42,7 +43,8 @@ object SummarySection extends JsonUtils {
       LeafExpr(path + "continueLabel", t.continueLabel) ++
       LeafExpr(path + "fields", t.fields) ++
       LeafExpr(path + "includeIf", t.includeIf) ++
-      LeafExpr(path + "pdf", t.pdf)
+      LeafExpr(path + "pdf", t.pdf) ++
+      LeafExpr(path + "excludeFromPdf", t.excludeFromPdf)
```

Comment:
This `excludeFromPdf` array doesn't contribute to any expression evaluation so we should not put this line here, to avoid https://github.com/hmrc/gform/pull/1152/files#r1579143668